modules/afsocket/afsocket-dest.c

```diff
@@ -207,6 +233,49 @@ afsocket_dd_start_reconnect_timer(AFSocketDestDriver *self)
   iv_timer_register(&self->reconnect_timer);
 }
 
+static inline void
+_copy_persist_entry_value(PersistState *state, PersistEntryHandle from, PersistEntryHandle to, gsize size)
```

Comment:
Optional: This would look nice as a PersistState API function, although a lot bigger refactors are needed to fully support the version conversion from PersistState API side...